shell/libretro/option.cpp

```diff
@@ -3,151 +3,152 @@
-Option<bool> SerialPTY("");
-Option<bool> UseReios(CORE_OPTION_NAME "_hle_bios");
-
-Option<bool> OpenGlChecks("", false);
-Option<bool> FastGDRomLoad(CORE_OPTION_NAME "_gdrom_fast_loading", false);
-Option<bool> RamMod32MB(CORE_OPTION_NAME "_dc_32mb_mod", false);
-
-//Option<std::vector<std::string>, false> ContentPath("");
-//Option<bool, false> HideLegacyNaomiRoms("", true);
-
-// Network
-
-Option<bool> NetworkEnable("", false);
-Option<bool> ActAsServer("", false);
-OptionString DNS("", "dns.flyca.st");
-OptionString NetworkServer("", "");
-Option<int> LocalPort("", 0);
-Option<bool> EmulateBBA(CORE_OPTION_NAME "_emulate_bba", false);
-Option<bool> EnableUPnP(CORE_OPTION_NAME "_upnp", true);
-Option<bool> GGPOEnable("", false);
-Option<int> GGPODelay("", 0);
-Option<bool> NetworkStats("", false);
-Option<int> GGPOAnalogAxes("", 0);
-Option<bool> NetworkOutput(CORE_OPTION_NAME "_network_output", false);
-Option<int> MultiboardSlaves("", 0);
-Option<bool> BattleCableEnable("", false);
-Option<bool> UseDCNet(CORE_OPTION_NAME "_dcnet", false);
-OptionString ISPUsername("", "flycast1");
-
-// Maple
-
-Option<int> MouseSensitivity("", 100);
-Option<int> VirtualGamepadVibration("", 20);
-
-std::array<Option<MapleDeviceType>, 4> MapleMainDevices {
-	Option<MapleDeviceType>("", MDT_None),
-	Option<MapleDeviceType>("", MDT_None),
-	Option<MapleDeviceType>("", MDT_None),
-	Option<MapleDeviceType>("", MDT_None),
-};
-std::array<std::array<Option<MapleDeviceType>, 2>, 4> MapleExpansionDevices {
-	Option<MapleDeviceType>("", MDT_None),
-	Option<MapleDeviceType>("", MDT_None),
-
-	Option<MapleDeviceType>("", MDT_None),
-	Option<MapleDeviceType>("", MDT_None),
-
-	Option<MapleDeviceType>("", MDT_None),
-	Option<MapleDeviceType>("", MDT_None),
-
-	Option<MapleDeviceType>("", MDT_None),
-	Option<MapleDeviceType>("", MDT_None),
-};
-
-} // namespace config
+	Option<bool> AutoLatency("");
+
+	OptionString AudioBackend("", "auto");
+	Option<bool> VmuSound(CORE_OPTION_NAME "_vmu_sound", false);
+
+	// Rendering
+
+	RendererOption RendererType;
+	Option<bool> UseMipmaps(CORE_OPTION_NAME "_mipmapping", true);
+	Option<bool> Widescreen(CORE_OPTION_NAME "_widescreen_hack");
+	Option<bool> SuperWidescreen("");
+	Option<bool> ShowFPS("");
+	Option<bool> RenderToTextureBuffer(CORE_OPTION_NAME "_enable_rttb");
+	Option<bool> TranslucentPolygonDepthMask("");
+	Option<bool> ModifierVolumes(CORE_OPTION_NAME "_volume_modifier_enable", true);
+	IntOption TextureUpscale(CORE_OPTION_NAME "_texupscale", 1);
+	IntOption MaxFilteredTextureSize(CORE_OPTION_NAME "_texupscale_max_filtered_texture_size", 256);
+	Option<float> ExtraDepthScale("", 1.f);
+	Option<bool> CustomTextures(CORE_OPTION_NAME "_custom_textures");
+	Option<bool> DumpTextures(CORE_OPTION_NAME "_dump_textures");
+	Option<int> ScreenStretching("", 100);
+	Option<bool> Fog(CORE_OPTION_NAME "_fog", true);
+	Option<bool> FloatVMUs("");
+	Option<bool> OnlyShowVMUA1(CORE_OPTION_NAME "_only_show_vmu_a1", false);
```

Comment:
This option isn't exposed in the libretro core (and it's not needed) so it's better to not give it an setting name:
```
Option<bool> OnlyShowVMUA1("");
```